.pipelines/versionSetting.ps1

```diff
@@ -58,3 +58,12 @@ $imageResizerContextMenuAppManifestReadFileLocation = $imageResizerContextMenuAp
 $imageResizerContextMenuAppManifest.Package.Identity.Version = $versionNumber + '.0'
 Write-Host "ImageResizerContextMenu version" $imageResizerContextMenuAppManifest.Package.Identity.Version
 $imageResizerContextMenuAppManifest.Save($imageResizerContextMenuAppManifestWriteFileLocation);
+
+# Set FileLocksmithContextMenu package version in AppManifest.xml
+$fileLocksmithContextMenuAppManifestWriteFileLocation = $PSScriptRoot + '/../src/modules/FileLocksmith/FileLocksmithContextMenu/AppxManifest.xml';
```

Comment:
this seems unrelated to the change described herein - should it be a separate PR?

Reply:
Whatever regression happened here seems to have been caused by our changes, which is weird.
Anyway, this is something done for the other tier 1 context menus and not this one.
Since it was only bringing issues with this PR, we're just including it here.